### DISCUSSION TITLE
[CRITICAL] [jp.tiarap.org] Certificate hash not found

### DISCUSSION THREAD

**Discussion by MojtabaFaal** [DNSCrypt/dnscrypt-proxy#1803] [Q&A] [answered]
Hi
What is this **CRITICAL** error?

`[2021-08-03 19:33:59] [CRITICAL] [jp.tiarap.org] Certificate hash [e57ce52d15106da4ecfd9e11fda5fd1d1e0a6be012241884d362ea1d4af464f7] not found`

`[2021-08-03 19:38:38] [CRITICAL] [jp.tiarap.org] Certificate hash [e57ce52d15106da4ecfd9e11fda5fd1d1e0a6be012241884d362ea1d4af464f7] not found`

I receive this **CRITICAL** error yesterday and today
What should I do ?

In addition :
I also deleted the following files but the error still remains :
- public-resolvers.md
- public-resolvers.md.minisig
- relays.md
- relays.md.minisig

**Comment by jedisct1** [comment, marked as answer]
This server's is using Cloudflare. And unfortunately, Cloudflare frequently moves IPs and certificates around, making it difficult to reliably use hashes.

The hashes have been removed from that server.

**Reply by MojtabaFaal** [reply]
thanks
